src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
+    jasmineGraphHashMapCentralStore->loadGraph();
+    graphDBMapCentralStores.insert(std::make_pair(graphIdentifier, *jasmineGraphHashMapCentralStore));
+    instance_logger.log("###INSTANCE### Loading central Store : Completed", "info");
+}
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string partitionID = (data);
-            partitionID = Utils::trim_copy(partitionID, " \f\n\r\t\v");
-            instance_logger.log("Received Partition ID: " + partitionID, "info");
+void JasmineGraphInstanceService::loadInstanceDuplicateCentralStore(
+    std::string graphId, std::string partitionId,
+    std::map<std::string, JasmineGraphHashMapDuplicateCentralStore> &graphDBMapDuplicateCentralStores) {
+    std::string graphIdentifier = graphId + +"_centralstore_dp_" + partitionId;
+    JasmineGraphHashMapDuplicateCentralStore *jasmineGraphHashMapCentralStore =
+        new JasmineGraphHashMapDuplicateCentralStore(atoi(graphId.c_str()), atoi(partitionId.c_str()));
+    jasmineGraphHashMapCentralStore->loadGraph();
+    graphDBMapDuplicateCentralStores.insert(std::make_pair(graphIdentifier, *jasmineGraphHashMapCentralStore));
+}
 
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
+JasmineGraphHashMapCentralStore JasmineGraphInstanceService::loadCentralStore(std::string centralStoreFileName) {
+    instance_logger.log("###INSTANCE### Loading Central Store File : Started " + centralStoreFileName, "info");
+    JasmineGraphHashMapCentralStore *jasmineGraphHashMapCentralStore = new JasmineGraphHashMapCentralStore();
+    jasmineGraphHashMapCentralStore->loadGraph(centralStoreFileName);
+    instance_logger.log("###INSTANCE### Loading Central Store File : Completed", "info");
+    return *jasmineGraphHashMapCentralStore;
+}
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string workerList = (data);
-            workerList = Utils::trim_copy(workerList, " \f\n\r\t\v");
-            instance_logger.log("Received Worker List " + workerList, "info");
-
-            std::vector<string> workerSockets;
-            stringstream wl(workerList);
-            string intermediate;
-            while (getline(wl, intermediate, ',')) {
-                workerSockets.push_back(intermediate);
-            }
+std::string JasmineGraphInstanceService::copyCentralStoreToAggregator(std::string graphId, std::string partitionId,
+                                                                      std::string aggregatorHost,
+                                                                      std::string aggregatorPort, std::string host) {
+    char buffer[128];
+    std::string result = "SUCCESS";
+    std::string centralGraphIdentifier = graphId + +"_centralstore_" + partitionId;
```

Comment:
```suggestion
    std::string centralGraphIdentifier = graphId + "_centralstore_" + partitionId;
```

Reply:
Fixed this and all similar occurrences of `+ +` in next PR

Reply:
Ok